docs/advanced-solidus/returns.mdx

```diff
@@ -0,0 +1,117 @@
+---
+sidebar_position: 8
+---
+
+# Returns
+
+This guide will go through what Solidus offers to manage the return of
+purchased items: the compensation of their cost in different ways or the
+exchange with other products.
+
+Sometimes things don't go as expected during an online transaction. Items can
+get damaged in their way, or the wrong product is shipped by error. Other
+times, companies have flexible devolution policies to remove friction in the
```

Comment:
Yeah, and I completely agree with the recommendation. However, I forgot to report why I was still keeping one instance of the "devolution" naming. My idea is to confirm that we're talking about the same topic to users that could be more familiar with that terminology. Also, according to the Diátaxis guides about explanations:

> It’s documentation that approaches a topic from a higher perspective, and **from different angles**.

However, if you think that's going to be too confusing, I'm entirely okay to remove it altogether.

> * I know that your plan is to integrate this content with the inline Ruby documentation but, just by going through the "Returns" guide, I've had a hard time understanding the entire flow of the returns system. For instance, the guide doesn't mention eligibility validators or refund amount calculators, even though those concepts are present in the how-tos.
> 
> * How are we planning to keep track of which guides have been written in compliance with Diátaxis and which ones have not? I would like to have a scalable system that helps other contributors understand how to approach each guide, in case they need to update it or move it around.
> 
> * IIRC, it's possible to implement custom reimbursement methods. Should we write a how-to for that as well?

Reply:
@waiting-for-dev I've never heard anyone working on the technical/business aspects of an online store say "devolution", everyone uses "return", so I would go with that. 🙂

Reply:
Fair enough. Changed!